spec/rspec/expectations/configuration_spec.rb

```diff
@@ -47,7 +45,13 @@ class << rspec_dup; undef configuration; end
       end
 
       context 'on an interpreter that does not provide BasicObject', :uses_should, :unless => defined?(::BasicObject) do
-        before { RSpec::Expectations::Syntax.disable_should(Delegator) }
+        around do |example|
+          in_sub_process_if_possible do
```

Comment:
`in_sub_process`/`in_sub_process_if_possible` is not safe to use in an `around` hook -- it makes the example _always_ pass.   Try adding literally anything (even something like `adfkjsadfsa` to the first line of the example) and you'll see it passes.

That's because `in_sub_process` communicates pass/fail to the parent process by re-raising an exception, but in an `around` hook, the example failure exception has already been handled by RSpec, and the execution result state gets updated--but that happens in the short-lived child process, and the parent process is never made aware of it.